src/service/Importer.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs');
+const path = require('path');
+const csv = require('csvtojson');
+const csvSync = require('csvjson');
+
+class Importer {
+  constructor(watcher, pathToDataDir) {
+    this.watcher = watcher;
+    this.pathToCsvData = pathToDataDir;
+
+    this.init();
+  }
+
+  static import(pathToCsvFile) {
+    return csv().fromFile(pathToCsvFile);
+  }
+
+  static importSync(pathToCsvFile) {
+    if (!fs.existsSync(pathToCsvFile)) {
+      return 'File not found';
```

Comment:
Polymorphic interfaces should be avoided as they bring a lot of confusion and often are source of subtle bugs. Please, make sure that returned result is consistent and is always of the same time. Currently, it is a string in one case and an object in other.

Reply:
Fixed by returning object instead of string.
Better solution for me is throwing an error,  but in this case the simpler solution is also fine.

Reply:
> Better solution for me is throwing an error, but in this case the simpler solution is also fine.

Yes, might be done this way as well